lib/archethic/metrics/metric_client.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ArchEthic.Metrics.MetricClient do
+  @moduledoc """
+  gensever
+  Visit dif
+  """
+  use GenServer
+  @process_name __MODULE__
+  @default_state %{
+    counter: 0
+  }
+  def start_link(_init_arg) do
+    GenServer.start_link(__MODULE__, @default_state, name: @process_name)
+  end
+
+  def init(_state) do
+    {:ok, @default_state}
```

Comment:
also the default state can be put directly.
Module attributes are useful when you want something at compilation time and or reused in the module

Reply:
noted